_data/roadmap.yml

```diff
@@ -1,60 +1,60 @@
 - year: 2014
   accomplishments:
     - name: Launched on Bitcointalk
-      date: April 18, 2014
+      date: 18 April 2014
```

Comment:
>April 18, 2014
>18 April 2014

Converting from American English to British is a trivial moot point. I see no reason to change something that can be so easily converted back to its original format while keeping functionality intact. Why make these changes?